metabase/metabase
# Metadata editor for models

This PR introduces metadata editing for models (ex datasets). This is a feature branch containing both FE and BE changes broken down into sub-projects:

* #19194
* #19206
* #19291
* #19425
* #19379
* #19724 

You can learn more about the feature in the [project doc](https://www.notion.so/metabase/Allow-for-the-creation-and-usage-of-models-7090fa371e4d49d4a6f3260fcb0b4ab2#80ac40ef6ece478bb37930efe04d2879) and/or from sub-projects PR descriptions above.

### To Verify

Go through the "To Verify" sections from the PRs above. Beware: some of them still use "dataset" instead of "model"

### Demo

https://user-images.githubusercontent.com/17258145/149556601-f50d2b16-4b8d-4f6a-bcd0-c80705122007.mov
